docs/integration/alerts-alertops-integration.md

```diff
@@ -0,0 +1,25 @@
+title: AlertOps Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with AlertOps - a real-time, intelligent, automated incident management platform. Use them together for seamless infrastructure and application monitoring integration with alert-messaging, and begin sending webhook notifications to reach the appropriate team by using proper workflows, escalation policies and schedules. 
+
+Based on your ruleset, incidents can be automatically opened and closed, depending on whether Sematext reports a problem or a recovery. 
+
+The above scenario and scope for integration is due to the fact that AlertOps has a very flexible and simple API/Webhook configuration feature that can be leveraged with Sematext’s monitoring and alerting capabilities.
+
+## **In AlertOps**
+
+Log into your [AlertOps account](https://alertops.com/) and configure *inbound* integration between AlertOps and Sematext APM, Log Management, Tracing, Experience, and Synthetics platform. Also check [AlertOps's integration guide](https://help.alertops.com/en/articles/5554660-sematext) for more integration details.
```

Comment:
Sematext APM, Log Management, Tracing, Experience, and Synthetics platform ==> Sematext